protocol/x/subaccounts/types/keys.go

```diff
@@ -11,6 +11,9 @@ const (
 
 // State
 const (
-	// SubaccountKeyPrefix is the prefix to retrieve all Subaccount
+	// SubaccountKeyPrefix is the prefix to retrieve all Subaccounts
 	SubaccountKeyPrefix = "SA:"
+	// NegativeTncSubaccountSeenAtBlockKey is the store key that stores the last
+	// block a negative TNC subaccount was seen in state.
+	NegativeTncSubaccountSeenAtBlockKey = "NegSA:"
```

Comment:
```suggestion
	NegativeTncSubaccountSeenAtBlockKey = "LastNegSA:"
```

Reply:
Let's just keep it short. We wouldn't need to look at this unless we are using `iaviewer` so it's fine for this to be less human readable

Reply:
Agreed with Jay!